components/automate-chef-io/content/docs/iam-v1-overview.md

```diff
@@ -47,7 +48,7 @@ Policies can contain wildcards to match a range of values.
 
 ## Structure
 
-A policy is made up of three essential components: the **subjects**, which is
+A policy has three essential components: the **subjects**, which is
```

Comment:
Does a policy have more than these 3 components? If not then perhaps:
```suggestion
A policy has three components: the **subjects**, which is
```

Reply:
done